_config.yml

```diff
@@ -60,8 +60,6 @@ title: "Apache Arrow"
 description: "Apache Arrow is the universal columnar format and multi-language toolbox for fast data interchange and in-memory analytics. It specifies a standardized language-independent column-oriented memory format for flat and nested data, organized for efficient analytic operations on modern hardware. It also provides computational libraries and zero-copy streaming messaging and interprocess communication. Languages currently supported include C, C++, C#, Go, Java, JavaScript, MATLAB, Python, R, Ruby, and Rust."
 url: https://arrow.apache.org
 logo: /img/logo.png
-twitter:
-  username: ApacheArrow
```

Comment:
This could be used in layout files (for example like: `href="https://twitter.com/{{ site.twitter.username }}"`) but instead we are using full links so this is not needed.